### ISSUE TITLE
Misleading log: DPU instead of core

### ISSUE THREAD

**Issue by jakubklimek** [UnifiedViews/Core#279]
![image](https://cloud.githubusercontent.com/assets/718827/5940473/6658d636-a70b-11e4-9f30-3af6e7c98a2f.png)
This error is actually reported by core. However, in the log, it is marked as T-RdfToFiles

**Comment by tomas-knap** [comment]
Yes, but it is produced by Core as it prepares inputs for T-RdfToFiles. If we do not introduces any DPU instance, it may be misleading, cannot be filtered out so that the pipeline designer knows it happens before T-RdfToFiles. 

What is the suggested solution?

**Comment by tomas-knap** [comment]
To reproduce: 
1) Place t-rdfToFiles to the pipelines, with no input edge. 
2) Run the pipeline.

**Comment by tomas-knap** [comment]
The problem is that the log is a bit confusing, as the pipeline designer does not know nothing about input field "inRdfData", which is actually variable holding input RDF data unit in the DPU's code.

**Comment by tomas-knap** [comment]
Relates with https://github.com/UnifiedViews/Core/issues/278
